styles.css

```diff
@@ -0,0 +1,190 @@
+body {
+  background-color: #191765;
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+  width: 100%;
+}
+
+@media only screen and (min-width: 375px) {
+  /* Header Styling */
+  .navbar {
+    display: flex;
+    color: white;
+    position: relative;
+    top: 43px;
+    justify-content: space-around;
+    width: calc(100% - 100px);
+  }
+
+  .header-name {
+    color: white;
+    text-decoration: none;
+    font-size: 18px;
+  }
+
+  .hamburger-menu {
+    position: relative;
+    left: 103px;
+  }
+
+  /* Introduction Styling */
+
+  .introduction-image {
+    background-image: url(images/Ellipse18.svg);
+    margin: 0;
+    margin-left: 0;
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 225px;
+    position: absolute;
+  }
+
+  .introduction {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    min-height: 100px;
+  }
+
+  .introduction-name {
+    color: #7d73fc;
+    font-size: 48px;
+    position: relative;
+    text-align: center;
+    margin-top: 94px;
+    left: 46px;
+  }
+
+  .soft-developer {
+    color: white;
+    position: relative;
+    top: -37px;
+    font-size: 32px;
+    left: 38px;
+  }
+
+  .intro-paragraph {
+    margin: 0;
+    color: #c8c8d4;
+    position: relative;
+    top: -25px;
+    font-size: 20px;
+    line-height: 2;
+    right: -38px;
+  }
+
+  .intro-line {
+    position: relative;
+    left: 53px;
+    top: 24px;
+  }
+
+  /* Experience Styling */
+
+  .experience {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    position: relative;
+  }
+
+  .experience-image {
+    background-image: url(images/Ellipse19.svg);
+    background-repeat: no-repeat;
+    width: 23%;
+    height: 144px;
+    position: absolute;
+    right: -82px;
+    top: 82px;
+  }
+
+  .language {
+    display: flex;
+    justify-content: space-evenly;
+    margin-top: 59px;
+  }
+
+  .list-text {
+    color: white;
+    list-style: none;
+    margin-left: -37px;
+    line-height: 2;
+  }
+
+  .skills-align {
+    margin-left: 57px;
+  }
+
+  .lang-align {
+    margin-right: 37px;
+  }
+
+  .experience-heading {
+    color: #9490ff;
+  }
+
+  .experience-line {
+    position: relative;
+    right: -13px;
+  }
+
+  /* Styling Social Media */
+  .social-media {
+    display: flex;
+    list-style: none;
+    text-decoration: none;
+    justify-content: space-evenly;
+  }
+
+  .work {
+    display: flex;
+    color: #a2a3ae;
+    align-items: center;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .recent-work {
+    color: #9490ff;
+    margin-left: 29px;
+    font-size: 40px;
+  }
+
+  .recent-work-text {
+    color: white;
+  }
```

Comment:
- [ ] Kindly use `display: grid` to the main container of the projects as per the requirement.